### ISSUE TITLE
Separation of concerns.

### ISSUE THREAD

**Issue by vivekimsit** [transferwise/comparison-components#16]
I think it would be good idea to separate logic from presentation. For context:

https://medium.com/@learnreact/container-components-c0e67432e005#.6fv524za6

wdyt?

**Comment by oliverviljamaa** [comment]
I agree – in addition to the better maintainability, it would allow us to use the dumb component directly in some cases.
For example, to check whether to show a `Compare price` button on TransferWise front page, we already make the call and could pass the comparison data directly to the component rather than having it make an extra call for it (which is the case now).

**Comment by Tankenstein** [comment]
Yup, i'm planning to work on this (been saying this forever, though) for a revamp of a page.

**Comment by vivekimsit** [comment]
@Tankenstein sounds cool, it would be great if I can contribute to the new branch,
